wsclient/src/main/java/org/hpccsystems/ws/client/platform/Workunit.java

```diff
@@ -690,7 +690,7 @@ void fullRefresh(boolean includeGraphs, boolean includeResults, boolean includeS
             HPCCWsWorkUnitsClient wsWorkunitsClient = platform.getWsClient().getWsWorkunitsClient();
             try
             {
-                WorkunitWrapper wi = wsWorkunitsClient.getWUInfo(info.getWuid(), includeResults, includeGraphs, includeSourceFiles, includeApplicationValues, false, false, false, false, false);
+                WorkunitWrapper wi = wsWorkunitsClient.getWUInfo(info.getWuid());
```

Comment:
This PR might stil be WIP, but this change could change behavior, the includeResults, includeGraphs, includeSourceFiles, includeApplicationValues params would be ignored.

Reply:
Correct, we deprecated in candidate-7.10.0, the alternative is getWUInfo(WSInfoRequestWrapper)
So we need to call it like this:
WUInfoRequestWrapper wuinforeq = new WUInfoRequestWrapper();
            	wuinforeq.setWuid(info.getWuid());
            	wuinforeq.setIncludeResults(includeResults);
            	wuinforeq.setIncludeGraphs(includeGraphs);
            	wuinforeq.setIncludeSourceFiles(includeSourceFiles);
            	wuinforeq.setIncludeApplicationValues(includeApplicationValues);
            	wuinforeq.setIncludeDebugValues(false);
            	wuinforeq.setIncludeExceptions(false);
            	wuinforeq.setIncludeVariables(false);
            	wuinforeq.setIncludeXmlSchemas(false);
            	wuinforeq.setIncludeTimers(false);
            	
                WorkunitWrapper wi = wsWorkunitsClient.getWUInfo(wuinforeq);

Reply:
@Michael-Gardner bump above comment